WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeListActivity.java

```diff
@@ -298,7 +304,16 @@ public void onEventMainThread(PublicizeEvents.ActionCompleted event) {
         }
         reloadDetailFragment();
 
-        if (!event.didSucceed()) {
+        if (event.didSucceed()) {
+            Map<String, Object> analyticsProperties = new HashMap<>();
+            analyticsProperties.put("service", event.getService());
+
+            if (event.getAction() == ConnectAction.CONNECT || event.getAction() == ConnectAction.RECONNECT) {
+                AnalyticsUtils.trackWithSiteDetails(Stat.PUBLICIZE_SERVICE_CONNECTED, mSite, analyticsProperties);
+            } else if (event.getAction() == ConnectAction.DISCONNECT) {
+                AnalyticsUtils.trackWithSiteDetails(Stat.PUBLICIZE_SERVICE_DISCONNECTED, mSite, analyticsProperties);
```

Comment:
While disconnecting an account, the breakpoint I hit on this line was hit twice.  Is that expected?  Will it skew our tracking of `wpandroid_publicize_service_disconnected`?

Reply:
I'm not sure why it's hit twice, but I only see it being recorded once in Tracks.

Reply:
I added a log message just before the analytics statement in the `else if` branch.  The message was only logged once so I guess it was just a debugging issue.